client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -199,3 +167,119 @@ const PlanTypeSelector: React.FunctionComponent< Props > = ( props ) => {
 };
 
 export default PlanTypeSelector;
+
```

Comment:
I am intrigued!
Is there an internal practice of moving code into emotion? Or is there a specific reason this code is moved into js?